src/components/Expander/Expander.less

```diff
@@ -67,4 +67,35 @@
 	&-content {
 		overflow: hidden;
 	}
+
+	// kind
+	&-with-emphasis {
+
+		.lucid-Expander-header {
+			background: @color-backgroundColor;
+			border: @border-lightBorder;
+			padding: @size-standard;
+			display: flex;
+
+			.lucid-Expander-text {
+				font-size: 15px;
```

Comment:
I asked him to leave this at 15px actually. It _should_ have a `@font-size-medium` or something to handle this one. We need to improve our font size variables.